userguide/tutorials/reference-time-examples.adoc

```diff
@@ -0,0 +1,91 @@
+= Reference Time/Fixed Offset Timezone
+
+In this document, we look a little bit deeper into the concepts of reference time and fixed offset timezone.
+
+== Reference Time
+
+Each account in Kill Bill has a *reference time* associated with it. It can be specified at the time of creating the account. If not specified, it defaults to the account creation date time. Kill Bill uses the reference time to convert a `LocalDate` to a `DateTime` and vice-versa. So for example, if the invoice for an account is due on a particular date, the reference time is used to determine the time of the day at which the invoice will actually be generated. Note that the reference time is always considered to be in UTC.
+
+Let us take a look at a few examples of how the reference time influences invoice generation.
+
+=== Example 1
+
+* Set the time to `2023-01-01T11:00`.
```

Comment:
For all your examples, you need to specify a fully qualified date: e.g: `2023-01-01T11:00:00.000Z`

Reply:
Done